mantidproject/mantid
# Fix user doc errors and warnings before starting 6.7 release note work

**Description of work**

Doc errors and warnings have built up once more. This PR is fixing them before work begins on the 6.7 release notes. Most errors were incorrect references. I have also moved lingering release notes from PR's that were opened before 6.6 released (so release notes went under `6.6`) but merged after.

Most importantly, there were a couple reference to a non-existent doc for the new loader `LoadILLLagrange` added here #35202.
This algorithm needs to have a doc written for it, currently clicking '?' on the algorithm interface takes you to a 404 error page.

There are still some doc warnings hanging around but all are concerned with the 6.7 release notes not having new lines at the end of their bullet points etc. Obviously, these will all be sorted out when they're moved to the main files.

**To test:**

- Delete `build/docs`
- Build `docs-html` target
- Check there are no warnings other than minor positional warnings that will be sorted by the main release note file creation.
- Check there are no errors in the build.

Fixes #35473



---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
